lib/operations/logic/notEqual.js

```diff
@@ -0,0 +1,7 @@
+function notEqual(a, b) {
+  // eslint-disable-next-line eqeqeq
+  return a != b;
+}
+
+notEqual.code = '!=';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`!=` is an existing operator